crml/transaction-payment/src/lib.rs

```diff
@@ -445,7 +441,6 @@ mod tests {
 	type Balances = pallet_balances::Module<Runtime>;
 	type System = frame_system::Module<Runtime>;
 	type TransactionPayment = Module<Runtime>;
-	type TestFeeExchange = TransactionFeeExchange<Runtime>;
```

Comment:
Compare this deleted line with the added one at 413. I would say this one is tidier and less error-prone. Any changes to Runtime would be followed automatically.